src/app/shared/components/paragraph-oe-quote/paragraph-oe-quote.component.html

```diff
@@ -0,0 +1,14 @@
+<figure class="ecl-blockquote">
```

Comment:
mmh..this is not using the eui component: `<figure eclBlockquote
        author="President Juncker">
    An interconnected grid will help deliver the ultimate goal of the Energy Union, to
    ensure affordable, secure and sustainable energy, and also growth across the EU.
    <picture eclBlockquotePicture>
        <img eclBlockquoteImage
             src="https://inno-ecl.s3.amazonaws.com/media/examples/example-image-square.jpg"
             alt="Image alternative text" />
    </picture>
</figure>`